evm/src/omni-bridge/contracts/BridgeTypes.sol

```diff
@@ -44,7 +44,8 @@ library BridgeTypes {
         string token,
         string name,
         string symbol,
-        uint8 decimals
+        uint8 decimals,
+        uint8 originDecimals
```

Comment:
Do we need `originDecimals`? Since we have `token` we can look it up on Near

Reply:
It is a little bit complicated to retrieve the origin decimals due to async specific of the cross calls on Near, so by adding this we can reduce the number of callbacks on Near side